### PR TITLE
Attach casedb to model not view

### PR DESCRIPTION
**Pull request by benrudolph**
@emord http://manage.dimagi.com/default.asp?186654#1042337
this attaches the casedb to the model instead of the view so it'll appear on the case view and detail view

@proteusvacuum

### CODE REVIEW

File: corehq/apps/cloudcare/static/cloudcare/js/backbone/cases.js [hunk 1]
```diff
@@ -244,11 +244,11 @@ cloudCare.CaseListView = Backbone.View.extend({
     },
     appendItem: function (item) {
         var self = this;
+        item.set('casedb', self.caseList.casedb)
```
Comment (dimagimon) [inline corehq/apps/cloudcare/static/cloudcare/js/backbone/cases.js]: Missing semicolon.

File: corehq/apps/cloudcare/static/cloudcare/js/backbone/cases.js [hunk 2]
```diff
@@ -103,8 +103,8 @@ cloudCare.Details = Backbone.Model.extend({
 cloudCare.caseViewMixin = {
     lookupField: function (field) {
         var parentCase;
-        if (isParentField(field) && this.options.casedb) {
-            parentCase = this.options.casedb.get(this.model.get('indices').parent.case_id);
+        if (isParentField(field) && this.model.get('casedb')) {
+            parentCase = this.model.get('casedb').get(this.model.get('indices').parent.case_id);
```
Comment (proteusvacuum) [inline corehq/apps/cloudcare/static/cloudcare/js/backbone/cases.js]: interesting. What happens if the index identifier is not `parent`? (I'm assuming that's what `.parent` is referring to).

Comment (proteusvacuum) [inline corehq/apps/cloudcare/static/cloudcare/js/backbone/cases.js]: Actually, ignore me. I think you can only adjust identifiers for Advanced Forms, and I don't think you can use them in cloudcare.

**Comment by proteusvacuum** [comment]
:+1:

File: corehq/apps/cloudcare/static/cloudcare/js/backbone/cases.js (hunk 2 above)
Comment (benrudolph) [inline corehq/apps/cloudcare/static/cloudcare/js/backbone/cases.js]: yeah we have very limited support for showing any kind of relationships in the case details/list

**Pull request merged**